docker/startup.sh

```diff
@@ -15,4 +15,4 @@ cd algorithmic-efficiency
 git pull
 
 chmod a+x docker/scripts/startup.sh
-docker/scripts/startup.sh -f $FRAMEWORK -u $GIT_URL -d $DATASET -s $SUBMISSION_PATH -t $TUNING_SEARCH_SPACE -e $EXPERIMENT_NAME -w $WORKLOAD
+docker/scripts/startup.sh -f $FRAMEWORK -s $SUBMISSION_PATH -t $TUNING_SEARCH_SPACE -e $EXPERIMENT_NAME -w $WORKLOAD
```

Comment:
is it possible to avoid having this file, and just put these three commands inside the dockerfile? it's confusing to have docker/startup.sh and docker/scripts/startup.sh

Reply:
so Dockerfile can only run scripts that it has locally which is this script 

this script clones git repo and pulls to get a remotely controlled startup script, otherwise it's too annoying to rebuild image everytime just to make it run a different command, this setup I found useful to quickly iterate and have control

Reply:
once we know exactly what docker images should run on startup we can remove the git controlled startup script

Reply:
spoke offline, simplified docker setup by eliminating extra startup.sh